src/templates/pycontw-2020/contents/en/covid-19/guidelines.html

```diff
@@ -1,5 +1,5 @@
 {% extends 'contents/_base.html' %}
-{% load static %}
```

Comment:
Build fail. I think you need this line since this is used in `<img src="{% static ...`

Reply:
okay , I updated it.